packages/outdated-browser/src/index.ts

```diff
@@ -19,53 +18,59 @@ const style = `
     z-index: 9999;
     width: 100%;
     display: flex;
-    gap: 16px;
-    background: rgba(242, 86, 72, 0.95);
-    flex-direction: column;
+    gap: 24px;
+    background: #ff8c69;
     align-items: center;
-    text-align: center;
-    padding: 16px;
+    padding: 16px 60px 16px 16px;
     color: white;
     font-family: 'Open Sans', 'Segoe UI', sans-serif;
+    box-shadow: 0px 8px 40px 10px #151A2D5C;
   }
 
-  #outdated svg {
-    max-width: 300px;
+  @media (max-width: 768px) {
+    #outdated {
+      flex-direction: column;
+      align-items: unset;
+    }
   }
 
-  #outdated a {
-    color: white;
-    border: 1px solid white;
-    border-radius: 4px;
-    background: transparent;
-    padding: 10px 20px;
-    text-decoration: none;
-    text-transform: uppercase;
-    transition: all .3s ease-in;
+  #outdated p {
+    font-size: 12px;
   }
 
-  #outdated a:hover {
-    color: rgb(242, 86, 72);
-    background: white;
+  #outdated svg {
+    max-height: 28px;
+    width: fit-content;
   }
 
   #outdated button {
     color: white;
     background: transparent;
-    font-size: 32px;
+    font-size: 28px;
+    line-height: 28px;
     position: absolute;
     border: 0;
-    right: 60px;
-    top: 10px;
+    right: 20px;
+    top: 16px;
     cursor: pointer;
   }
 `
 
 declare const SUPPORTED_BROWSERS: string
 
+const STORAGE_KEY = '__outdated'
+const ignore = sessionStorage.getItem(STORAGE_KEY) || 'false'
```

Comment:
Why not local storage?

Reply:
Yes, that's intended :)

Each time he will close and reopen its browser